data/labs.yaml

```
@@ -24,6 +24,11 @@ labs:
         - Rachid
         - Guerraoui
       email: rachid.guerraoui@epfl.ch
+    description: >
+      he Distributed Computing Lab focuses currently on Scalable Implementations 
```

It looks like the prettier pass ate the first letter of several descriptions?
```suggestion
      The Distributed Computing Lab focuses currently on Scalable Implementations 
```